README.md

```diff
@@ -110,7 +110,6 @@ Anyone can add an entry to the table, and entries should include the:
 | ---- | -------- | -------- | --------- | ---------------------- | ------------ | ----------- |
 | 40Km | 900M | 50HZ | 10mW | No | Snipes | https://www.youtube.com/watch?v=0QWN9qWoSYY |
 | 33Km | 2.4G | 250HZ | 100mW | No | Snipes | https://www.youtube.com/watch?v=GkOCT17a-DE |
-| 30Km | 900M | 50HZ | 1W | No | Snipes | https://www.youtube.com/watch?v=SbWvFIpVkto |
```

Comment:
Might as well remove the entry below as well? (10km 2.4 250HZ).

Reply:
agreed